### DISCUSSION TITLE
Strange diagonal pattern at first layer with Ender3S1 F4

### DISCUSSION THREAD

**Discussion by franz134** [mriscoc/Ender3V2S1#246] [Q&A]
My setup: 
- Ender 3 S1 (F4) with Ender3S1-F4-20220716.bin 
- printing via Octoprint
- installed exactly according to the instructions (leveling, pid....)
I'm new to this alternative firmware and had newer a version before this mid-year update.

Initially after the installation I experienced "loud" noises (probably from the steppers) at some diagonal movements (e.g. homing) but not all. Also there are noises when printing rounded corners (skirt lines), but I didn't mind because I read #229 that this should increase the accuracy and I had no problem with a 50% calibration cube.

**Now my issue is that I get ugly strange diagonal pattern perpendicular to the filling line orientation, which also interfere with each other layer by layer.**
I have no clue what the problem is because I already set the original/suggested motion settings from https://github.com/mriscoc/Ender3V2S1/issues/229#issuecomment-1186583002 (and tried even lower) and had newer a problem with the original firmware.
_My next step would be to try 20220527 and if this works I'll need to learn how to recompile the current version with ADAPTIVE_STEP_SMOOTHING enabled I guess... might this be the issue, because I also noticed that the printer is slower than Cura estimated with the original firmware and with 20220716 its approximately on point_

![IMG_20220722_163844](https://user-images.githubusercontent.com/98423516/180608012-1b3e5613-9305-4f32-a4be-25e887c3cfb9.jpg)
- first layer
![IMG_20220722_164939](https://user-images.githubusercontent.com/98423516/180608024-e775a371-6be2-4faa-8695-4b29d2324d46.jpg)
- second layer
![IMG_20220722_170748](https://user-images.githubusercontent.com/98423516/180608040-7d7a9671-6427-4644-9d5b-64959014f7ba.jpg)
- interference

**Comment by MarkforU** [comment]
I'm using Octoprint. I'm even using UBL Firmware and 81 point mesh. Either your mesh isn't loading or by the looks of it, your might be a bit close on your z-offset. My routine goes. Use Tram Wizard, get all 4 corners locked in. Next do 81 point bed level and save that to slot 0 and to slot 1 to be on safe side. Then set your z-offset to 0. Do a Z-offset Home and Probe. Take it down to where nozzle is just about to touch bed. Install a SD card with a Gcode file of stl of a 40x40x1mm square to be printed. Launch that from the SD card through printer, make sure z-offset is looking good and is nice flat and smooth lines coming out whether you want to try at .2 or .28 would actually be my first recommendation so you know you're not too close to bed. Oh, hope you're running CR/BL-Touch. It makes things much easier, but that first layer in picture has nothing to do with this firmware by far...that's user problem. If you are using a touch, you better make sure your startup code, which I use Cura, is correct and I would be doing UBL Startup and 4 point check before print I'm case bed corner changes. 
Oh, after your first print layer is good, just same startup code use again but this time print from Cura through Octoprint instead of memory card. You offset is set and it should be perfect. 
Sorry just looked at pictures again you have CR-Touch...you should be perfect with UBL and startup.

**Reply by franz134** [reply]
you're right, my z-offset was about 0.09mm to close. Sorry this is my first printer and I was astonished because I probably hit the correct offset spot on with the original firmware... maybe luck.
Thanks for your detailed routine, but I'm wondering why I don't have "slot 0 & 1" only "save or continue" after my typical 7x7 mesh (but might be normal since I'm not using UBL)

**Reply by mriscoc** [reply]
Slot 0..n is only for UBL version of the firmware
